pkg/csi/service/common/commonco/k8sorchestrator/topology.go

```diff
@@ -88,8 +93,12 @@ type nodeVolumeTopology struct {
 	csiNodeTopologyWatcher *cache.ListWatch
 	// k8sClient is a kubernetes client.
 	k8sClient clientset.Interface
-	//k8sConfig is the in-cluster config for client to talk to the api-server.
+	// k8sConfig is the in-cluster config for client to talk to the api-server.
 	k8sConfig *restclient.Config
+	// clusterFlavor is the cluster flavor.
+	clusterFlavor cnstypes.CnsClusterFlavor
+	// useCSINodeIdFeatureEnabled indicates whether the cluster flavor is enabled or not.
+	useCSINodeIdFeatureEnabled bool
```

Comment:
can you use a better name for `useCSINodeIdFeatureEnabled`?

Reply:
We are using consistently the feature name as 'use-csinode-id`. I we pick a different name, I think it will be consuming.